snap/snapcraft.yaml

```diff
@@ -0,0 +1,231 @@
+name: yubioath-desktop
+adopt-info: yubioath-desktop
+icon: resources/icons/com.yubico.yubioath.svg
+base: core18
+grade: devel
+confinement: strict
+
+plugs:
+  ykman-config:
+    interface: personal-files
+    write:
+      - $HOME/.ykman
+apps:
+  pcscd:
+    command: usr/sbin/pcscd --foreground --auto-exit
+    daemon: simple
+    common-id: com.yubico.pcscd
+    plugs:
+      - hardware-observe
+      - network
+      - network-bind
```

Comment:
Offline conversation: double check that network plugs are needed

Reply:
It seems to be needed, I get this when removing them from `pcscd`. It looks like I can drop them from the `yubioath-desktop` part though!
```
$ /snap/yubioath-desktop/current/usr/sbin/pcscd --foreground
00000000 pcscdaemon.c:698:main() cannot create /run/snap.yubioath-desktop/pcscd.pid: Permission denied
00000071 winscard_msg_srv.c:148:InitializeSocket() Unable to bind common socket: Permission denied
00000010 pcscdaemon.c:723:main() Error initializing pcscd.
```

Reply:
:+1: